### PR TITLE
Adds logging to the antimatter and groups all the engines into one log

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

- Adds antimatter on/off and injection rate logging
- Groups all the engine logs into one category 

## Why It's Good For The Game

Being unable to track antimatter via logs has some griefing potential. 
Also log organization.

## Changelog
:cl:
admin: added logging for antimatter
admin: grouped singulo, antimatter and supermatter logs into one
/:cl:

### CODE REVIEW

File: code/__DEFINES/logging.dm [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #define INVESTIGATE_RECORDS			"records"
 #define INVESTIGATE_SINGULO			"singulo"
 #define INVESTIGATE_SUPERMATTER		"supermatter"
+#define INVESTIGATE_ANTIMATTER		"antimatter"
```
Comment (Archanial) [inline code/__DEFINES/logging.dm]: Just put it in supermatter log and rename it tbh

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/__DEFINES/logging.dm (hunk 1 above)
Comment (ivanmixo) [reply code/__DEFINES/logging.dm]: Yeah, was thinking about that.

**Comment by Sinestia** [comment]
It might be alright to throw in a message_admins when they create the console to say that it exists, since we're also told when the SM (and I believe a tesla/singulo) is created/started for the first time. 


Otherwise I agree with arch that this would fit well in a common log with supermatter, include singuloth too because it has its own investigate tab which it honestly probably doesn't need.

**Comment by ivanmixo** [comment]
> It might be alright to throw in a message_admins when they create the console to say that it exists, since we're also told when the SM (and I believe a tesla/singulo) is created/started for the first time.
> 
> Otherwise I agree with arch that this would fit well in a common log with supermatter, include singuloth too because it has its own investigate tab which it honestly probably doesn't need.

Sure, I'll just group them all into an engines log.

File: code/modules/power/antimatter/control.dm [hunk 2]
```diff
@@ -239,6 +240,9 @@
 
 /obj/machinery/power/am_control_unit/proc/toggle_power(powerfail = 0)
 	active = !active
+	investigate_log("turned [active ? "ON" : "OFF"] by [usr ? key_name(usr) : "outside forces"] at [AREACOORD(src)]", INVESTIGATE_ENGINES)
+	message_admins("Antimatter turned [active ? "ON" : "OFF"] by [usr ? ADMIN_LOOKUPFLW(usr) : "outside forces"] in [ADMIN_VERBOSEJMP(src)]")
+	log_game("Antimatter turned [active ? "ON" : "OFF"] by [usr ? "[key_name(usr)]" : "outside forces"] at [AREACOORD(src)]")
```
Comment (Archanial) [inline code/modules/power/antimatter/control.dm]: Double logging bad, just log it in engines + message

**Review by Archanial — CHANGES_REQUESTED** [review]
Please make changelog say that singulo and sm investigates were merged

File: code/modules/power/antimatter/control.dm (hunk 2 above)
Comment (ivanmixo) [reply code/modules/power/antimatter/control.dm]: This is how the particle accelerator does it

Comment (Archanial) [reply code/modules/power/antimatter/control.dm]: Log game seems unnecessary in both cases

**Review by Archanial — APPROVED** [review]
:+1:

**Review by utilbird — APPROVED** [review]

**Pull request merged**